torch2jax/__init__.py

```diff
@@ -95,6 +94,13 @@ def __torch_function__(cls, func, types, args=(), kwargs=None):
     raise NotImplementedError(f"Torchish.__torch_function__: {func}")
 
   # fmt: off
+  @property
+  def device(self):
+    try:  # Attempt to get the device from the value's device.
+      return torch.device(self.value.device.type.replace("gpu", "cuda"))
+    except Exception:  # otherwise, assume CPU, e.g. abstrace arrays, tpu arrays
+      return torch.device("cpu")
```

Comment:
hmm, i think you've successfully sold me on making everything "cpu" here... why change to this? In general, dealing with devices across the jax/torch boundary seems like messy business, so i'm a bit more hesitant when picking this api. i especially want to avoid complexity. open to ideas, just want to make sure we get it right

now that i write this comment, would it also make sense to have `torch.device(<anything>)` return "cpu"? that way everything is consistent

if going this route, why is the `.replace("gpu", "cuda")` necessary?

Reply:
Let's unify with "cpu" then. So far I can't think of an example how this could fail.